server/core/templates/credentials_update_partial.html

```diff
@@ -147,7 +147,8 @@ <h4>UNIX Password</h4>
                     <div class="mt-2 pt-2 border-top">
                         <button class="btn btn-danger"
                             hx-post="/ui/api/cu_cancel"
-                            hx-target="#main">Discard Changes</button>
+                            hx-boost="false"
+                        >Discard Changes</button>
```

Comment:
```suggestion
                            hx-boost="false">
                                Discard Changes
                            </button>
```
Doesn't look like we have a consistent formatting style for HTML. Might be good to standardize so it's not whatever the dev (or reviewer) feels like

Reply:
Yep, I think we should do this is a separate issue to get that sorted. I know I'm a culprit here too.